modules/user/user.service.js

```diff
@@ -280,7 +283,7 @@ class UserService {
       throw new UserInputError(WRONG_CREDENTIALS, { statusCode: 400 });
     }
 
-    const token = generateToken(user._id, user.email);
+    const token = await generateToken(user._id, user.email);
```

Comment:
Refactor please, to not have an `await` for `generateToken` since it is a sync operation.

Reply:
fixed